finalPrep/modZeroHero.js

```diff
@@ -0,0 +1,176 @@
+// Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
+
+// Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Super Osito"
+var specialAbility = "Vacuum Mouth"
+
+
+// Declare two variables - greeting AND catchphrase
+//   greeting should be assigned to a string that uses concatenation to include the heroName
+//   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "Hi, I'm your friendly, neihborhood " + heroName
+
+var catchphrase = `Looks like a job for my ${specialAbility}!`
+
+console.log(greeting);
+console.log(catchphrase);
+
+
+// Declare two variables - power AND energy - set to integers
+var power = 5
+var energy = 10
+
+// Declare two variables - fullPower AND fullEnergy
+//   fullPower should multiply your current power by 500
+//   fullEnergy should add 150 to your current energy
+
+var fullPower = (power * 500)
+var fullEnergy = (energy + 150)
+console.log(fullPower);
+console.log(fullEnergy);
+
+// Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = false
+var identityConcealed = true
+
+// Declare two variables - archEnemies AND sidekicks
+//   archEnemies should be an array of at least 3 different enemy strings
+//   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["Vacuum Cleaner", "Roomba", "Broom"]
+var sidekicks = ["Zapatito", "Teddy", "Charlie"]
+
+console.log(archEnemies);
+console.log(sidekicks);
+
+// Print the first sidekick to your console
+console.log(sidekicks[0])
+
+// Print the last archEnemy to the console
+console.log(archEnemies[2])
+// Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("Fire Alarm");
+
+// Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
+
+// Remove the first sidekick from the sidekicks array
+archEnemies.shift();
+
+// Print the sidekicks array to console to ensure you added a new sidekick
+console.log(archEnemies);
+// Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
+//   - dangerLevel should be an integer
+//   - saveTheDay should be a string a hero would say once they save the day
+//   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  console.log(assessSituation)
```

Comment:
I would expect that invoking this function would trigger an error here - the computer will not know the meaning of the `console.log(assessSituation)` because that variable is not defined.